charts/rancher-monitoring/100.1.1+up19.0.3/values.yaml

```diff
@@ -415,6 +415,23 @@ hardenedNodeExporter:
       - effect: "NoSchedule"
         operator: "Exists"
 
+## Rancher Monitoring
+##
+
+rancherMonitoring:
+  enabled: true
```

Comment:
default enabled, can be disabled to specifically force this not to be deployed.

Even if this is enabled, if `selector` is `{}` and no Rancher deployment can be detected, it will be effectively the same as `enabled: false`.